samples/detect.v2beta1.js

```diff
@@ -0,0 +1,822 @@
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+
+  const fullName = knowledgeBaseFullName;
+
+  client
+    .deleteKnowledgeBase({
+      name: fullName,
+    })
+    .then(responses => {
+      const result = responses[0];
+      console.log(`Name: ${result.name}`);
+      console.log(`displayName: ${result.displayName}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END dialogflow_delete_knowledge_base]
+}
+
+function createDocument(
+  projectId,
+  knowledgeBaseFullName,
+  documentPath,
+  documentName
+) {
+  // [START dialogflow_create_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+  //const documentPath = `path of the document you'd like to add, e.g. https://dialogflow.com/docs/knowledge-connectors`;
+  //const documentName = `displayed name of your document in knowledge base, e.g. myDoc`;
+
+  var request = {
+    parent: knowledgeBaseFullName,
+    document: {
+      knowledgeTypes: [`FAQ`],
+      displayName: documentName,
+      contentUri: documentPath,
+      source: `contentUri`,
+      mimeType: `text/html`,
+    },
+  };
+
+  client
+    .createDocument(request)
+    .then(console.log(`Document created`))
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_create_document]
+}
+
+function listDocuments(projectId, knowledgeBaseFullName) {
+  // [START dialogflow_list_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+
+  client
+    .listDocuments({parent: knowledgeBaseFullName})
+    .then(responses => {
+      var resources = responses[0];
+      console.log(
+        `There are ${resources.length} documents in ${knowledgeBaseFullName}`
+      );
+      resources.forEach(r => {
```

Comment:
```js
resources.forEach(resource => {
  console.log(` KnowledgeType: ${resource.knowledgeType}`);
```

Do not use abbreviated variable names, even as block/closure var names, unless it's truly needed to stay < 80 characters and for things to look nice

Reply:
Note: `err` is conventional so it's used everywhere

Reply:
**Not a blocker**